### PR TITLE
[JIT] Remove dead store in quantization_patterns.h

### PR DESCRIPTION
**Pull request by SplitInfinity**
Test Plan: Continuous integration.

Differential Revision: D22259209

### CODE REVIEW

File: torch/csrc/jit/passes/quantization/quantization_patterns.h [hunk 1]
```diff
@@ -136,7 +136,6 @@ QuantFusionInfo getBinaryOpScalarFusionInfo(
   std::string op_pattern =
       getInputTensorQParamOpPattern(op_name, extra_op_args);
 
-  const auto& extra_op_arg_list = getExtraArgList(extra_op_args);
```
Comment (jerryzh168) [inline torch/csrc/jit/passes/quantization/quantization_patterns.h:139]: isn't this used in the next line?

**Comment by SplitInfinity** [comment]
I requested review before checking tests and then immediately unrequested, sorry about that. Let me figure out what's going on. This dead store was flagged by an internal tool.

File: torch/csrc/jit/passes/quantization/quantization_patterns.h (hunk 1 above)
Comment (SplitInfinity) [reply torch/csrc/jit/passes/quantization/quantization_patterns.h:139]: I requested review before checking tests and then immediately unrequested, sorry about that. Let me figure out what's going on. This dead store was flagged by an internal tool.

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization/quantization_patterns.h:139]: maybe it's related to `const auto&`, you can try changing to `auto`

**Review by jerryzh168 — APPROVED** [review]
I see, thanks!

**Pull request closed without merging**